src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
https://www.w3schools.com/cssref/css_default_values.asp
here you can find all default css rules for all tags